app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -404,6 +396,31 @@ abstract class BaseBrowserFragment : Fragment(), BackHandler, SessionManager.Obs
         }
     }
 
+    private fun getContextMenuCandidates(context: Context, view: View): List<ContextMenuCandidate> {
```

Comment:
Can you make this an abstract function and move the implementations to `BrowserFragment` and `ExternalAppBrowserFragment`?